packages/installer/exe/package.json

```diff
@@ -1,20 +0,0 @@
-{
-  "name": "@electron-forge/installer-exe",
-  "version": "6.0.0-beta.67",
-  "description": "EXE file installer for Electron Forge",
-  "repository": "https://github.com/electron-userland/electron-forge",
-  "author": "Samuel Attard",
-  "license": "MIT",
-  "main": "dist/InstallerExe.js",
-  "typings": "dist/InstallerExe.d.ts",
-  "devDependencies": {
-    "mocha": "^9.0.1"
-  },
-  "engines": {
-    "node": ">= 14.17.5"
-  },
-  "dependencies": {
-    "@electron-forge/installer-base": "6.0.0-beta.67",
-    "open": "^8.1.0"
```

Comment:
You should be able to get rid of `open` from `package.json` (and thus `yarn.lock`)

Reply:
[3fdd2fc](https://github.com/electron-userland/electron-forge/pull/2958/commits/3fdd2fca652187cc81d3c98a4b67beb9020a9899)